PROJECT_CHARTER.md

```diff
@@ -0,0 +1,72 @@
+# WebdriverIO Charter
+
+## Section 0: Guiding Principles
+
+The WebdriverIO project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1: Scope
+
+The project tries to provide users a variety of building blocks to create a highly customizable, maintainable and scaleable framework to build and run end-to-end (e2e) tests for web and mobile applications. While it is focused on running these tests in accordance with the [WebDriver](https://w3c.github.io/webdriver/) standard, it also leverages other interfaces to provide as much control of the automated user agent as possible.
+
+With that, the project offers not only the ability of automating user agents using open standards, it also enforces a common set of good practices that help to avoid pitfalls. These practices, along with solutions provided by the WebdriverIO community, serve to resolve well-defined problem spaces in dynamic and complex world of e2e testing.
+
+### 1.1: In-scope
+
+- automation of modern web browser
+- automation of mobile devices (web as well as native applications)
```

Comment:
maybe place a reference to Appium?

Reply:
WebdriverIO is not limited to Appium and I am afraid it would be out of scope of this list to include the names of all possible drivers.